README.rst

```diff
@@ -8,7 +8,7 @@ Let's Encrypt App for Django
    :target: https://codecov.io/gh/urda/django-letsencrypt/branch/master
 
 :code:`django-letsencrypt` will allow you to add, remove, and update any
-`ACME challenge <https://letsencrypt.github.io/acme-spec/>`_ objects you may
+`ACME challenge <https://tools.ietf.org/html/draft-ietf-acme-acme/>`_ objects you may
```

Comment:
I would prefer if we linked to this URL that is listed at the old endpoint:

```
https://github.com/ietf-wg-acme/acme/
```

As it is quoted from:

```
NOTE: This Repository is Deprecated

This repository is not active and does not accurately reflect what Let's Encrypt currently implements. It's retained only for history.

All new work happens here: https://github.com/ietf-wg-acme/acme/ 
```

Reply:
You will also need to rebase on to the `develop` branch for me.